src/administration/integrations/blackfire.md

```diff
@@ -29,7 +29,7 @@ runtime:
               server_token: '692203ae8755da6b57b8161d3f20dd1be71502f77adebf3363d164033d74d29b'
 ```
 
-Push your changes to your Platform environment to enable Blackfire:
+Push changes to your Platform environment to enable Blackfire as follows:
```

Comment:
I believe the "your" is still appropriate here.  Or perhaps "the".  Some word is needed here.